Documentation/building.md

```diff
@@ -22,10 +22,13 @@ Then you should be able to set the ARCH as above
 ## Building manually
 
 1. Make sure you have required dependencies installed on your machine.
-    * On Ubuntu, run `sudo apt-get install linux-libc-dev golang gcc`.
+    * On Ubuntu, run `sudo apt-get install linux-libc-dev golang gcc`. 
+      If the golang version installed is not 1.7 or higher. Download the newest golang and install manully.
+      To build the flannel.exe on windows, mingw-w64 is also needed. Run command `sudo apt-get install mingw-w64`
```

Comment:
Do you mean flanneld.exe?

Reply:
Yeah, it's a typo error. 
Meanwhile, for Fedora/Redhat, I think the mingw-w64 is also needed, but I only test on Ubuntu. Maybe someone who is in charge of the document can improve this section too.